keys/kali/kali.key

```diff
@@ -0,0 +1,1365 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
The key is called `Kali Linux Repository <devel@kali.org>`, so rename the file into `kali-linux-repository.key` by analogy with other keys.